### DISCUSSION TITLE
public access for other people

### DISCUSSION THREAD

**Discussion by domidetres** [Athou/commafeed#1545] [Q&A]
hey there,
is there a way to provide an access to my configured feeds as a read only with mutliple users?
I want to have a central news-database for some users which are centrally configured.
I am quite new to commafeed and configured all feeds in my admin account.
Thanks for helping! BR Dominic

**Comment by Athou** [comment]
Seems similar to [this discussion](https://github.com/Athou/commafeed/discussions/1537).
CommaFeed is a personal reader and was not thought with that kind of use-case in mind though.
